.github/workflows/tests_mergin_media_sync.yaml

```diff
@@ -0,0 +1,76 @@
+name: Tests for Mergin Maps Media Sync
+
+on:
+  push:
+    paths:
+    - "test/**"
+    - "**.py"
+    - ".github/workflows/tests_mergin_media_sync.yaml"
+
+env:
+  TEST_MERGIN_URL: https://test.dev.merginmaps.com
+  TEST_API_USERNAME: test_media_sync
```

Comment:
do we have this guy (and workspace) on test.dev? Also if server gets redeployed from scratch (it could anytime) - we need to make sure to have such account.